src/gg.regression.unity.bots/Runtime/Scripts/GenericBots/Experimental/GameSpeedup.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using UnityEngine;
+
+namespace RegressionGames.GenericBots.Experimental
+{
+    /// <summary>
+    /// Configures the game to run as fast as possible for faster training/search.
+    /// The speedup is active as soon as this object is constructed.
+    /// When the Dispose() method is invoked, the speedup is disabled.
+    /// The user of this class needs to regularly invoke the Update() method to ensure the speedup
+    /// is retained.
+    /// </summary>
+    public class GameSpeedup : IDisposable
```

Comment:
For some reason I feel like this could be a good small micro blog post (i.e. How to Speed Up Your Unity Game)... I'm gonna write that down

Reply:
Very cool, I would have benefited from that blog post! I now learned that there is to it than just setting timescale.